examples/gcs/README.md

```diff
@@ -20,9 +21,33 @@ resources:
     class: basic-admin
 ```
 
+## Infrastructure setup
+The workload service account will be automatically assigned to the necessary role with the selected policy.
+
+```mermaid
+graph TD;
+  gcs["Google Cloud Storage"]
+  policy["Google Cloud IAM Policy"]
+  role["Google Cloud IAM Role"]
+  subgraph EKS Cluster
```

Comment:
We have EKS here in a few, places should be GKE.

Reply:
Done!